platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/jasperdb/VirtualDataSourceJasperDB.java

```diff
@@ -152,37 +154,37 @@
      */
     private final boolean isLongKeyMode;
 
-    /** In memory off-heap store for path to disk location, this is used for both internal hashes store. */
-    private final LongList pathToDiskLocationInternalNodes;
+    /** In memory off-heap store for path to disk location. This is used for both node types. */
+    private final LongList hashPathToDiskLocation;
 
     /** In memory off-heap store for path to disk location, this is used by leave store. */
```

Comment:
"In memory off-heap store for path to disk location for leaves" (and "for hashes" above).

Reply:
✅